ArcGISRuntimeSDKQt_QMLSamples/Scenes/DisplayScenesInTabletopAR/DisplayScenesInTabletopAR.qml

```diff
@@ -0,0 +1,135 @@
+// [WriteFile Name=DisplayScenesInTabletopAR, Category=Scenes]
+// [Legal]
+// Copyright 2019 Esri.
+
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+// http://www.apache.org/licenses/LICENSE-2.0
+
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+// [Legal]
+
+import QtQuick 2.6
+import QtQuick.Controls 2.12
+import Esri.ArcGISRuntime 100.7
+import Esri.ArcGISExtras 1.1
+import Esri.ArcGISArToolkit 1.0
+
+Rectangle {
+    id: rootRectangle
+    clip: true
+    width: 800
+    height: 600
+
+    readonly property url dataPath: System.userHomePath + "/ArcGIS/Runtime/Data/mspk"
+    readonly property double sceneWidth: 800.0
+    readonly property double tableTopWidth: 1.0
+    property var philadelphiaScene: null
+
+    ArcGISArView {
+        id: arcGISArView
+        anchors.fill: parent
+        tracking: true
+        sceneView: sceneView
+        originCamera: Camera {}
```

Comment:
same question as with C++ - is supplying empty Camera and null locationDataSource required?